packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1231,6 +1233,7 @@ class RoundedDropDownButton<T> extends StatelessWidget {
               style: style,
               selectedItemBuilder: selectedItemBuilder,
               items: items,
+              focusColor: bgColor,
```

Comment:
Does this remove the focus effect on hover? If so, we shouldn't remove that. I originally filed a bug for this here but didn't have a minimal repro to attach: https://github.com/flutter/flutter/issues/116293

Reply:
It removes color effect, but mouse pointer changes, that looks to be enough.

Web:



https://user-images.githubusercontent.com/12115586/220485605-eb90485a-db55-4ca3-8333-376d0361a957.mov

Macos:



https://user-images.githubusercontent.com/12115586/220485628-33cc884b-07bb-4816-8d1f-86859453a7da.mov